src/tiled/commanddatamodel.cpp

```diff
@@ -433,6 +434,22 @@ bool CommandDataModel::dropMimeData(const QMimeData *data, Qt::DropAction, int,
     return false;
 }
 
+QKeySequence CommandDataModel::shortcut(const QModelIndex &index)
+{
+    const bool isNormalRow = index.row() < mCommands.size();
+
+    if (isNormalRow) {
+        return mCommands[index.row()].shortcut;
+    }
+    else
+        return QKeySequence();
```

Comment:
This implementation leads to a segmentation fault, because `isValid()` returns `true` when the last row is chosen when the model doesn't actually exist.